libs/dh/wholesale/feature-request-calculation/src/lib/dh-wholesale-request-calculation.html

```diff
@@ -0,0 +1,76 @@
+<!--
+@license
+Copyright 2020 Energinet DataHub A/S
+
+Licensed under the Apache License, Version 2.0 (the "License2");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<watt-card vater inset="0" *transloco="let t; read: 'wholesale.requestCalculation'">
+  <form [formGroup]="form" (ngSubmit)="requestCalculation()">
+    <vater-flex fill="vertical" gap="m">
+      <vater-stack align="flex-start" direction="column" gap="s">
+        <watt-dropdown
+          translate="wholesale.requestCalculation.processTypes"
+          [label]="t('processType')"
+          [formControl]="form.controls.processType"
+          [options]="progressTypeOptions"
+          dhDropdownTranslator
```

Comment:
issue: missing translations for some options

![image](https://github.com/Energinet-DataHub/greenforce-frontend/assets/1096332/3629a4c3-020b-4166-8a0c-06a0a1005a41)